install.py

```diff
@@ -192,6 +224,43 @@ def predict_candidate_git_tag_names(self):
             ]
         return tag_names
 
+    def do_git_clone(self):
+        if not Cmd.which('git'):
+            raise InstallError('git command not found. Install git.')
+
+        branch = None
+        if self.git_branch:
+            branch = self.git_branch
+        else:
+            branch = self.predict_git_commit_branch()
+
+        git_clone_cmd = 'git clone -b {branch} --depth=1 {repo_url}'.format(
+            branch=branch,
+            repo_url=self.RPM_GIT_REPO_URL,
+        )
+        Cmd.sh_e(git_clone_cmd)
+
+    def predict_git_commit_branch(self):
```

Comment:
Rename `git_commit_branch` to `git_branch`.
Rename `commit_branch` to `git_branch`.
method name and variable name.